### ISSUE TITLE
Missing alt text for user avatar in Accessibility Team Meeting Agenda post.

### ISSUE THREAD

**Issue by truptikanzariya** [WordPress/gutenberg#70726]
The image of the user avatar (Nazmul Hasan Robin) in the "Accessibility Team Meeting Agenda: July 08, 2025" post on the [Make WordPress Accessibility blog](https://make.wordpress.org/accessibility/) is missing appropriate alt text.

URL :  https://make.wordpress.org/accessibility/

**Steps to Reproduce:**

1. Visit the "Accessibility Team Meeting Agenda: July 08, 2025" post.

2. Locate the user avatar for Nazmul Hasan Robin in the post.

3. Observe that the alt text for the image is either missing or not properly defined.

**Expected Result:**

The image should have a meaningful alt text to provide an accessible experience for screen reader users. An example alt text could be "Avatar of Nazmul Hasan Robin."

**Suggested Fix:**
Please add alt text to the user avatar image to ensure the content is accessible to all users.

**Screen shot:**

<img width="1307" height="640" alt="Image" src="https://github.com/user-attachments/assets/31364f00-5cea-4495-96f8-a11e32bd9b02" />

**Comment by t-hamano** [comment]
Thanks for the report.

I think the HTML you mentioned is rendered by [the o2 plugin](https://github.com/Automattic/o2), not Gutenberg.

Please refer to the issue I submitted: https://github.com/Automattic/o2/issues/240
